layers/meta-resin-edison/conf/samples/local.conf.sample

```diff
@@ -34,6 +34,9 @@ DISTRO ?= "resin-systemd"
 # http://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#ref-classes-rm-work
 #INHERIT += "rm_work"
 
+# Remove the old image before the new one generated to save disk space
```

Comment:
Maybe you could reword the comment:

"Remove the old image before the new one generated to save disk space"

feels a bit weird. Maybe "is generated" is more appropriate